pkg/azure/aks/price_store.go

```diff
@@ -69,13 +96,26 @@ func NewPricingStore(parentContext context.Context, parentLogger *slog.Logger, s
 	return p, err
 }
 
-func (p *PriceStore) getPriceInfoFromVmInfo(vmInfo *VirtualMachineInfo) (float64, error) {
+func (p *PriceStore) getPriceBreakdownFromVmInfo(vmInfo *VirtualMachineInfo, price float64) *MachinePrices {
+	ratio, ok := cpuToCostRatio[vmInfo.MachineFamily]
+	if !ok {
+		p.logger.LogAttrs(p.context, slog.LevelInfo, "no ratio found for instance type, using default", slog.String("instanceType", vmInfo.MachineTypeSku), slog.String("instanceFamily", vmInfo.MachineFamily))
```

Comment:
nit: Split out each attribute to it's own line. It makes it much easier to read/grok.

Reply:
done